auctions/templates/addon/base.html

```diff
@@ -33,10 +33,19 @@
 
          <div id="widget-input-header">
              <a href="https://{{current_site.domain}}" target="_blank">
-                 <img id="widget-header-logo" src="{% static "img/codesy_300dpi_323x80.png" %}" width="161" height="40"></img>
+                <img id="widget-header-logo"
+                    src="{% static "img/codesy_300dpi_323x80.png" %}" width="161" height="40">
+                </img>
              </a>
          </div>
 
+        {% if active_issue %}
+            <div class="widget-icon" >
+                <i class="fi-lightbulb "></i>
```

Comment:
r+ on condition that @maxxcrawford re-visits this. 😜

Reply:
Also, we don't need to show this(?) if the user has their *own* bid on the issue.